topaz/objects/arrayobject.py

```diff
@@ -26,7 +26,7 @@ def lt(self, w_a, w_b):
         if self.space.is_kind_of(w_cmp_res, self.space.w_bignum):
             return self.space.bigint_w(w_cmp_res).lt(rbigint.fromint(0))
         else:
-            return self.space.int_w(w_cmp_res) < 0
+            return Coerce.int(self.space, w_cmp_res) < 0
```

Comment:
:heart_eyes:  Updated with the way! https://github.com/topazproject/topaz/commit/deabda38d7cc41f0b90c13ecd4bc278fde607527